### PR TITLE
provide option for controling rendering of model objects

### PR DESCRIPTION
**Pull request by seanpkps**
implementation for feature requested by issue #1230 

(it also contains some updates to the gulpfile which I found made development easier ... more details, see the closed pull request #1768)

### CODE REVIEW

**Comment by seanpkps** [comment]
@ponelat re: #1230 
(looking forward to having this integrated)

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     }
   ],
   "description": "Swagger UI is a dependency-free collection of HTML, JavaScript, and CSS assets that dynamically generate beautiful documentation from a Swagger-compliant API",
-  "version": "2.1.4",
+  "version": "2.1.5",
```
Comment (ponelat) [inline package.json]: Please let the admin bump versions

File: gulpfile.js [hunk 2]
```diff
@@ -124,15 +119,29 @@ gulp.task('copy', ['less'], function() {
     .src(['./src/main/html/**/*'])
     .pipe(gulp.dest('./dist'))
     .on('error', log);
+}
+gulp.task('dev-copy', ['dev-less', 'copy-local-specs'], _copy);
+
+gulp.task('copy-local-specs', function () {
```
Comment (ponelat) [inline gulpfile.js]: Quite useful, I'd imagine... +1

**Comment by ponelat** [comment]
@seanpkps thanks again for the work here.
Could you rebase? There should be the travis workaround merged, so that we can run the suite.
If that passes, I just need to verify the build works as expected (looks good, just need to double check :smile:)

**Comment by seanpkps** [comment]
@ponelat - rebased and tests passing (also removed the version number bump)

**Comment by ponelat** [comment]
looks good. 
@fehguy merge when you're ready, please :smile:

**Pull request merged**